cmd_arger.c

```diff
@@ -69,14 +69,13 @@ void cmd_arger_parse(CmdArgerDesc* optional_args, uint32_t optional_args_count,
 
 			//
 			// if we find --help, then just ignore everything, print the help message and exist the program
-			static char help_name[] = "help";
-			if (name_or_value_len == 4 && memcmp(name_or_value, help_name, sizeof(help_name)) == 0) {
+			if (name_or_value_len == 4 && memcmp(name_or_value, "help", sizeof("help")) == 0) {
```

Comment:
This may just be a matter of personal taste, it's a pretty minor issue.

There's no need (other than literal deduplication?) to make this copy.
If you just use the literals, the compiler can replace the string comparison with an integer alternative (see https://godbolt.org/z/nxcKeojYs), avoiding the need for a string in memory to begin with.